src/functionalTest/resources/json/bulkscan/version2/validation/expectedWarnings/invalidApplyingExecutor0EmailAddress.txt

```diff
@@ -0,0 +1 @@
+Second applicant email address (executorsApplying_0_applyingExecutorEmail) does not appear to be a valid email address
```

Comment:
Is this text gonna appear in the logs ? Would it always be applicable on Second email address. What if 3rd or 4th executer email is invalid and not the second ?

Reply:
This error message won't log, and it show on the XUI and the user can correct the OCR problem

For Second email address, this is only a function test case, add by me, all 2nd , 3rd & 4th executor email address are not mandatory,
if any executors email address is invalid, the relative error message will show

The email regex has already used by the primaryApplicantEmailAddress, solsSolicitorEmail & caveatorEmailAddress at OcrEmailValidator.java with the EmailValidationService.java before